generator.sh

```
@@ -158,46 +200,45 @@ SH_ECHO
-  ${endIf}
-${endSym}
+${fnName}${startSym}${if}${ifCond}${then}echo "${para}" | lolcat${else}echo "${para}"${endIf}${endSym}
 LOLCAT
 
     # echo.Reset to remove color code on output
     fnName="${fn}echo${dot}Reset${brackets}"
     cat << RESET >> "${tempDist}"
-${fnName}${startSym}
-  echo "\$${para}" | tr -d '[:cntrl:]' | sed -E "s/\\\\[((;)?[0-9]{1,3}){0,3}m//g" | xargs
-${endSym}
+${fnName}${startSym}${exec}echo "${para}" | tr -d ${trCntrl} | sed -E ${leftQuote}s/${escape}[((;)?[0-9]{1,3}){0,3}m//g${rightQuote} | xargs${endSym}
```

Shellcheck (0.11.0)

[warning] exec is referenced but not assigned (for output from commands, use "$(exec ...)" ).

(SC2154)